icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -117,185 +116,251 @@
     }
 
     /**
-     * Enumeration for SI prefixes, such as "kilo".
+     * Enumeration for SI and binary prefixes, e.g. "kilo-", "nano-", "mebi-".
      *
-     * @draft ICU 68
+     * @draft ICU 69
      * @provisional This API might change or be removed in a future release.
      */
-    public enum SIPrefix {
+    public enum MeasurePrefix {
 
         /**
          * SI prefix: yotta, 10^24.
          *
          * @draft ICU 68
          * @provisional This API might change or be removed in a future release.
          */
-        YOTTA(24, "yotta"),
+        YOTTA(24, "yotta", 10),
```

Comment:
I prefer instead of using the number. use a constant. 
Using constant is more illustrative and error-prone. it is easy to miss type a number.

for example: `MeasurePrefix.SI_BASE`

Reply:
This is also a public class - though, Java at least has private members. I hear what you're saying, but again: committee review. (At least I think it's harder to "mistype" numbers like 10 and 1024, it's tested in the unit tests, and client code doesn't need to be typing these numbers...)

Reply:
I'm in favor of just writing 10 here on the Java side.  This code will never need to be changed, and we are doing a review right now to make sure it is correct.

Reply:
thanks. I'm leaving this as just "10" and "1024", trusting conversion unit tests, prefix unit tests, and code review to ensure they're right.